# These two env vars are needed for running the ISL test harness
# and making it do the "right thing".
# bash --noprofile is documented.
# Not for upstream. They don't run their test harness the way we do.
--- codegen_test.sh.in	2013-09-13 10:23:28.000000000 -0700
+++ codegen_test.sh.in	2015-07-23 08:12:25.013762502 -0700
@@ -1,10 +1,15 @@
-#!/bin/sh
+#!/bin/bash --noprofile
+
+set -o posix
 
 EXEEXT=@EXEEXT@
 srcdir=@srcdir@
 
 failed=0
 
+export libtool_install_magic='not set'
+export libtool_execute_magic='not set'
+
 for i in $srcdir/test_inputs/codegen/*.in \
 		$srcdir/test_inputs/codegen/cloog/*.in \
 		$srcdir/test_inputs/codegen/omega/*.in \
